.gitignore

```diff
@@ -56,6 +54,19 @@ clients/php/test/Api/*
 !clients/php/test/Api/LocalesApiTest.php
 !clients/php/test/Api/UploadsApiTest.php
 
+clients/go/*.go
```

Comment:
We've discussed this here https://github.com/phrase/openapi/pull/427#discussion_r1356900271 and decided not to go this individual exclusion route. so, feel free to leave this file as is, and only `git add -f` files that need to be added.